app/api/attendees.py

```diff
@@ -0,0 +1,93 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+
+from app.models import db
+from app.models.order import Order
+from app.models.ticket_holder import TicketHolder
+from app.models.event import Event
+from app.models.ticket import Ticket
+from app.api.helpers.permissions import jwt_required, is_registrar
+from app.api.helpers.utilities import dasherize
+
+
+class AttendeeSchema(Schema):
+    """
+    Api schema for Session Model
+    """
+    class Meta:
+        """
+        Meta class for Session Api Schema
+        """
+        type_ = 'attendee'
+        self_view = 'v1.attendee_detail'
+        self_view_kwargs = {'id': '<id>'}
+        inflect = dasherize
+
+    id = fields.Str(dump_only=True)
+    firstname = fields.Str(required=True)
+    lastname = fields.Str()
+    email = fields.Str()
+    address = fields.Str()
+    city = fields.Str()
+    state = fields.Str()
+    country = fields.Str()
+    is_checked_in = fields.Boolean()
+    pdf_url = fields.Url(required=True)
+
+    order = Relationship(attribute='order',
+                                 self_view='v1.attendee_order',
+                                 self_view_kwargs={'id': '<id>'},
+                                 related_view='v1.order_detail',
+                                 related_view_kwargs={'order_id': '<id>'},
+                                 schema='OrderSchema',
+                                 type_='order')
+
+    ticket = Relationship(attribute='ticket',
+                                self_view='v1.attendee_ticket',
+                                self_view_kwargs={'id': '<id>'},
+                                related_view='v1.ticket_detail',
+                                related_view_kwargs={'ticket_id': '<id>'},
+                                schema='TicketSchema',
+                                type_='ticket')
+
+
+class AttendeeList(ResourceList):
+    """
+    List and create Attendees
+    """
+    def query(self, view_kwargs):
+        query_ = self.session.query(TicketHolder)
+        if view_kwargs.get('order_id') is not None:
```

Comment:
It can be done like this : `if view_kwargs.get('order_id'):` i.e. removing `is not None`.

Same goes for the next `if` statement.